action-server/tests/actions/test_action_initialize_daily_checkin.py

```diff
@@ -48,29 +57,37 @@
     }
 }
 
-NON_DEFAULT_REMINDER_VALUES = {
+NON_DEFAULT_REMINDER_ATTRIBUTES = {
+    FIRST_NAME_ATTRIBUTE: "Robin",
+    PROVINCE_ATTRIBUTE: NON_DEFAULT_PROVINCE,
+    AGE_OVER_65_ATTRIBUTE: True,
+    PRECONDITIONS_ATTRIBUTE: True,
+    HAS_DIALOGUE_ATTRIBUTE: True,
+}
+
+NON_DEFAULT_REMINDER_SLOTS = {
```

Comment:
What's the difference between attributes and slots here?

Reply:
Attributes are the name of proprieties of the reminder, slots are, well, slots... The mapping was direct before so there were errors that caused no problem, but renaming the province slot highlighted them. And I can't change the reminder property accordingly because it won't fit with what's already in the DB

Reply:
Well, what's in the DB can be changed, but I get why you didn't went this way.
But is there a way to use a `Reminder`/`Assessment` object here instead?